lib/connect.js

```diff
@@ -279,6 +280,11 @@ function connect(token, options) {
  *   are not provided.
  * @property {boolean} [dominantSpeaker=false] - Whether to enable the Dominant
  *   Speaker API or not. This only takes effect in Group Rooms.
+ * @property {boolean} [dscpTagging=false] - DSCP tagging allows you to request enhanced
+ * QoS treatment for RTP media packets from any firewall that the client may be behind.
+ * Setting this option to <code>true</code>, will request DSCP tagging for audio packets
+ * on supported platforms. Audio packets will be sent with DSCP header value set to (0xb8)
```

Comment:
@makarandp0 Nitpick: `on supported browsers (only Chrome supports this as of now). Audio packets...`?